src/ax/endpoint/validator.go

```diff
@@ -0,0 +1,47 @@
+package endpoint
+
+import (
+	"context"
+
+	"github.com/jmalloc/ax/src/ax"
+)
+
+// Validator wraps methods required to
+// perform message validation
+type Validator interface {
+	Validate(ctx context.Context, msg ax.Message) error
+}
+
+// DefaultValidators is the array of validators used by default
+// when sending a message in case none of user-defined validators
+// have been defined.
+var DefaultValidators = []Validator{
+	&BasicValidator{},
+}
```

Comment:
Not an array :P  Sometimes when I want generic language I'll use the word "set", like:

> DefaultValidators is the set of validators used to validate outgoing messages if no other set of validators is configured on the endpoint.

Reply:
Corrected :-)